src/options/options.html

```diff
@@ -1,21 +1,40 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
     <!-- <link rel="stylesheet" href="../forExtensionPages.css">
     <link rel="stylesheet" href="./style.css"> -->
+    <!-- Font Awesome CDN -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css"
```

Comment:
The delete icon is loading from here that's why it's not coming when I remove this line. I guess it should get this from the `.woff` files you have stored inside `font-awesome` folder in `assets`, right?